doc/configuration.markdown

```diff
@@ -124,4 +124,13 @@ which will act as the cache key generator:
 	end
 	
 For more options see the [Rack::Request documentation](http://rack.rubyforge.org/doc/classes/Rack/Request.html)
-	
+
+### `use_native_ttl`
+
+Passes on the expiration timestamp to the cache store. This may be necessary
+with some stores to keep them from filling up, e.g. if using a Redis backend and
```

Comment:
there is entity and meta store ... and there should be no reason
redis-rack-cache is needed, it should just be `store: Redis.new or
Rails.cache` and done :D
But that's for another PR ... just FYI since I thought you built your own
backend and had input to that idea.

On Tue, Feb 2, 2016 at 12:32 AM, Julien Letessier <notifications@github.com>
wrote:

> In doc/configuration.markdown
> <https://github.com/rtomayko/rack-cache/pull/117#discussion_r51537743>:
>
> > @@ -124,4 +124,13 @@ which will act as the cache key generator:
> >  	end
> >  	
> >  For more options see the [Rack::Request documentation](http://rack.rubyforge.org/doc/classes/Rack/Request.html)
> > -	
> > +
> > +### `use_native_ttl`
> > +
> > +Passes on the expiration timestamp to the cache store. This may be necessary
> > +with some stores to keep them from filling up, e.g. if using a Redis backend and
>
> you wrote your own redis backend
> I didn't. This PR is mainly about making Rack::Cache work correctly with
> the redis-store/redis-rack-cache
> <https://github.com/redis-store/redis-rack-cache> gem.
>
> want to link that in the readme
> Sure thing.
>
> FYI I kind of want to get rid of all these silly backends and just support
> a common read/write interface and be one with it
> Not sure what you mean? To my understanding Rack::Cache already has
> pluggable backends, as long as they're namespaced correctly, respond to a
> .resolve(uri) factory, and have instances that quack #exists?, #read, and
> #write?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rtomayko/rack-cache/pull/117/files#r51537743>.
>